src/C-DEngine/C-DViewModels/cde_HSI.cs

```diff
@@ -2171,14 +2173,28 @@ public override string ToString()
         /// Returns a Markup Version of ToString()
         /// </summary>
         /// <returns></returns>
-        public string ToMLString(bool BrAll = false)
+        public string ToMLString(bool brAll = false)
```

Comment:
I understand why you did this, but unfortunately changing a parameter of a public function could result in a break to API users that explicitly set this parameter during the function call. You could mark this as obsolete and request to use the override with "includeScope" but you cannot rename the parameter.